docker-compose.yml

```diff
@@ -46,7 +46,7 @@ services:
       - ~/.ssh:/root/.ssh
     restart: always
     extra_hosts:
-      - "docker-local.dev:172.18.0.1"
+      - "docker-local.localhost:172.18.0.1"
```

Comment:
will this cause issues with folks already setup with .dev? should we leave this as-is, and just update the README?

Reply:
I don't know how often people are pulling updates to this if they already have it set up. Either way, they should be able to fix this pretty easily if it does cause issues (I think it would likely only impact cron). I think best to have it correct going forward, so I vote leave this :)